helm/libero-reviewer/templates/deployment-continuum-auth.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "libero-reviewer.fullname" . }}-continuum-auth
+  labels:
+    app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+    helm.sh/chart: {{ include "libero-reviewer.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: client
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
really need this? Not useful to have a `replicaCount` that is the same for all Deployments, so just hardcode 1?